components/camel-kafka/src/test/java/org/apache/camel/component/kafka/integration/KafkaConsumerAsyncManualCommitIT.java

```diff
@@ -74,8 +83,6 @@ public void after() {
         if (producer != null) {
             producer.close();
         }
-        // clean all test topics
-        kafkaAdminClient.deleteTopics(Collections.singletonList(TOPIC));
```

Comment:
Why we need to remove these clean codes?

Reply:
In this case, we do want the data to remain in the topic between tests. If we delete the topic between test executions, then the 3rd test scenario will have no uncommitted data to fetch from.